distributed/client.py

```diff
@@ -1871,7 +1871,7 @@ async def _gather_remote(self, direct, local_worker):
 
         return response
 
-    def gather(self, futures, errors="raise", direct=None, asynchronous=None):
+    def gather(self, futures, errors="raise", direct=None, asynchronous=None, callback_timeout=None):
```

Comment:
I think we call this `timeout` elsewhere

```suggestion
    def gather(self, futures, errors="raise", direct=None, asynchronous=None, timeout=None):
```